### PR TITLE
Plugin and synchronisation

### PR DESCRIPTION
**Pull request by rjzondervan**
Enable the new plugin structure and with it the new Synchronisation routes

### CODE REVIEW

File: api/src/Subscriber/EndpointSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+use App\Entity\Endpoint;
+use App\Event\ActionEvent;
+use Doctrine\ORM\EntityManagerInterface;
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Response;
+use Ramsey\Uuid\Uuid;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class EndpointSubscriber implements EventSubscriberInterface
```
Comment (WilcoLouwerse) [inline api/src/Subscriber/EndpointSubscriber.php:15]: Missing some docblocks here in this subscriber

File: api/src/Subscriber/ZaaktypeSubscriber.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+use App\Entity\ObjectEntity;
+use App\Event\ActionEvent;
+use App\Repository\ObjectEntityRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Exception\BadRequestException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Serializer\Encoder\XmlEncoder;
+
+class ZaaktypeSubscriber implements EventSubscriberInterface
```
Comment (WilcoLouwerse) [inline api/src/Subscriber/ZaaktypeSubscriber.php:17]: Missing some docblocks here in this subscriber

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: api/src/Subscriber/EndpointSubscriber.php (hunk 1 above)
Comment (rjzondervan) [reply api/src/Subscriber/EndpointSubscriber.php]: Thanks for spotting this, these subscribers have been deprecated with the new plugin route of course, they have been deleted now.

File: api/src/Subscriber/ZaaktypeSubscriber.php (hunk 2 above)
Comment (rjzondervan) [reply api/src/Subscriber/ZaaktypeSubscriber.php]: Thanks for spotting this, these subscribers have been deprecated with the new plugin route of course, they have been deleted now.

**Review by smisidjan — APPROVED** [review]

**Pull request merged**